server/repository/src/db_diesel/form_schema_row.rs

```diff
@@ -80,15 +82,26 @@ impl<'a> FormSchemaRowRepository<'a> {
         FormSchemaRowRepository { connection }
     }
 
-    pub fn upsert_one(&self, schema: &FormSchemaJson) -> Result<(), RepositoryError> {
+    pub fn upsert_one(&self, schema: &FormSchemaJson) -> Result<i64, RepositoryError> {
```

Comment:
A little different from other sync examples where this inserts the FormSchemaJson struct rather than FormSchemaRow struct.

Wasn't entirely sure if my pattern was correct in this case. 

As seen in other parts of PR - inserting / changelog uses the FormSchemaJson struct